dist/api_data.py

```diff
@@ -789,6 +789,16 @@ def __ge__(self, other):
                 directory must already exist. If the value is not an absolute path, the path
                 is relative to the database home (see @ref absolute_path for more information)'''),
         ]),
+    Config('rollback_to_stable', '', r'''
+        rollback tables to an earlier point in time, discarding all updates to checkpoint durable
+        tables that have durable times more recent than the current global stable timestamp''',
+        type='category', subconfig=[
+            Config('threads', 4, r'''
+                maximum number of threads WiredTiger will start to help RTS. Each
+                RTS worker thread uses a session from the configured WT_RTS_MAX_WORKERS''',
+                min=0,
+                max=10),    # !!! Must match WT_RTS_MAX_WORKERS
```

Comment:
Looks like `WT_RTS_MAX_WORKERS` is not defined in the `mongodb-8.0` branch.

Reply:
This PR defined it [here](https://github.com/wiredtiger/wiredtiger/pull/11343/files#diff-f9996701c9940c39d54fa5894eb3c313250daddcce5c108bb49d0948945ebd10R87)

Reply:
@zunyiliu, am I correct?